scripts/v1-nxm/v1-cn-locked.js

```diff
@@ -0,0 +1,112 @@
+require('dotenv').config();
+const { inspect } = require('node:util');
+
+const deployments = require('@nexusmutual/deployments');
+const { Sema } = require('async-sema');
+const fs = require('fs');
+const { ethers } = require('hardhat');
+
+const { BigNumber } = ethers;
+
+const OUTPUT_FILE = 'v1-cn-locked-amount.json';
+const ROLE_MEMBER = 2;
+
+const getContractFactory = async providerOrSigner => {
+  return async contractName => {
+    const abi = deployments[contractName];
+    const address = deployments.addresses[contractName];
+    return new ethers.Contract(address, abi, providerOrSigner);
+  };
+};
+
+async function getMemberCN(memberId, mr, tc) {
+  try {
+    const [member, active] = await mr.memberAtIndex(ROLE_MEMBER, memberId);
+
+    if (!active) {
+      return { member, amount: '0' };
+    }
+
+    const { coverIds, lockReasons, withdrawableAmount } = await tc.getWithdrawableCoverNotes(member);
+    const memberLockReasons = await tc.getLockReasons(member);
+
+    const lockReasonIndexCover = {};
+    let coverIndex = 0;
+    for (const lockReason of lockReasons) {
+      const lockReasonIndex = memberLockReasons.indexOf(lockReason);
+      lockReasonIndexCover[lockReasonIndex] = BigNumber.from(coverIds[coverIndex++]).toString();
+    }
+
+    const sortedIndexes = Object.keys(lockReasonIndexCover).sort((a, b) => a - b);
```

Comment:
As far as I remember the withdraw function was picky regarding the order. Is this sorting done to solve that problem or is it just cosmetic?

Reply:
i believe it for the withdraw. this CN code is a direct copy from the [get-cn-locked.js](https://github.com/NexusMutual/smart-contracts/blob/bb65519f6ff1e12ec4dab0db6eb4a52e9caa2dae/scripts/v2-migration/get-cn-locked.js) migration script.